receiver/routereceiver/README.md

```diff
@@ -0,0 +1,41 @@
+# Route Receiver
+This receiver is used to receive telemetry routed from other pipelines.
+
+## Supported pipelines
+- Logs
+- Metrics
+- Traces
+
+## How It Works
+1. The user configures this receiver in a pipeline.
+2. The user configures a supported component to route telemetry to this receiver.
+
+## Configuration
+The route receiver does not have any configuration parameters. It simply receives telemetry from other components.
+
+### Example Config
+The following config is an example configuration of the route receiver. In this example, logs are collected from a file and sent to a log count processor. After each minute, the log counts are converted to metrics and sent to the route receiver in the metrics pipeline.
```

Comment:
Is it worth linking to the log count processor from here?

Reply:
Not yet. I think once we have some more components that use the route receiver, we include this list at the bottom of the readme. Maybe with a header that says "supported components."